AutoCollection/WebSnippet.ts

```diff
@@ -29,6 +33,7 @@ class WebSnippet {
 
         //TODO: quick fix for bundle error, remove this when npm is published
         WebSnippet._snippet = snippetInjectionHelper.webSnippet.replace("INSTRUMENTATION_KEY", client.config.instrumentationKey);
+        WebSnippet._statsBeats = client.getStatsbeat();
```

Comment:
If you instantiate this class multiple times you could be overriding the Statsbeat instance because you added it as static